tests/prmbackslash_2.prm

```diff
@@ -33,7 +33,7 @@ set Surface pressure\
 set Use years in out\
 put instead of seconds = \
 false  # default: true
-set Nonlinear solver\
+set Nonlinear solver \
```

Comment:
Yes, that is correct. The previous implementation could have the space either before the backslash, or at the beginning of the next line, but the Deal.II implementation trims leading whitespace on continued lines so it is necessary to have the space before the backslash in order to avoid the above being parsed as 'set Nonlinear solverscheme'